outputs/conferencebestpractices.md

```diff
@@ -17,8 +17,15 @@ The aim of this checklist is for it to be a non-exhaustive but illustrative refe
 - [ ] There is a published Diversity and Inclusion statement around any efforts and policies
 - [ ] Efforts are taken to make the event space able to be navigated with people with mobility issues
 
-
 ## Safety
 - [ ] Social activities associated with the event do not have a strong alcohol focus
 - [ ] Safety of participants using public transportation is taken into account in planning
 - [ ] There is a clear method for participants to avoid being in photography and any social media
+
+## Logistics
+- [ ] There are available methods to display one's pronoun (stickers, pre-printed on badge, write-in space); day-of event all attendees are encouraged to use them
+- [ ] Scent-free spaces are considered/designated or participants are asked in advance to limit scents
+- [ ] Space is set aside for those who need a quieter zone without much sensory stimulation
+- [ ] Designated space for those with mobility needs is created in rooms used by the conference
+- [ ] An ASL interpreter or live transcription is provided for all keynote/large-events (at minimum)
+- [ ] A mother's room or other designated space is made available
```

Comment:
I assume "mothers room" includes for breast feeding and/or lactation? Or am I wrong? Maybe we can say that in case someone may not know what this refers to?

Reply:
That was the idea. I'll clarify.